Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Powershell_Command_Execution.yml

```diff
@@ -0,0 +1,1481 @@
+id: silent-Suspicious Powershell Command Execution
+version: -1
+issilent: true
+name: silent-Suspicious Powershell Command Execution
+description: |-
+  This playbook handles the following alerts:
+
+  - PowerShell runs suspicious base64-encoded commands
+  - PowerShell suspicious flags
+  - Reverse shell using PowerShell
+  - PowerShell downloads files via BITS
+
+  Playbook Stages
+
+  Analysis:
+
+  - Analyze command-line for suspicious patterns
+  - Evaluate risk score of PowerShell commands
+  - Check for indicators of compromise (IOCs)
+    - If suspicious/malicious patterns detected:
+      - Execute Remediation
+
+  Investigation:
+
+  - Check process reputation
```

Comment:
- Check the process reputation.

Question: is this the order the investigation is performed? If so, should it should be a numbered list instead of a bulleted list